includes/class-block-patterns.php

```diff
@@ -104,6 +111,24 @@ public function add_taxonomies_on_insert_post( $post, $request ) {
 		}
 	}
 
+	/**
+	 * Localize conditional loading based on version check.
+	 *
+	 * @access public
+	 */
+	public function conditional_load_patterns() {
+		global $wp_version;
+		$version = str_replace( '-src', '', $wp_version ); // Strip '-src' from the version string. Messes up version_compare().
+
+		wp_localize_script(
+			'coblocks-editor',
+			'coblocksBlockPatterns',
+			array(
+				'patternsEnabled' => version_compare( $version, '5.5-beta', '>=' ),
```

Comment:
Let's utilize the built-in function [`is_wp_version_compatible`](https://developer.wordpress.org/reference/functions/is_wp_version_compatible/) here instead to simplify.